src/_posts/platform/deployment/2000-01-01-deploy-with-git.md

```diff
@@ -82,19 +82,38 @@ platform/app/2000-01-01-procfile %}).
 
 Once a deployment has succeeded, you can recover the code you pushed to Scalingo using:
 
-```
+```bash
 git clone --origin scalingo git@ssh.osc-fr1.scalingo.com:my-app.git
 ```
 
 This will create a copy of the Git history hosted in Scalingo repositories.
 It will also add the `scalingo` remote to simplify future pushes.
 
 {% note %}
-  Only Git operations will be available via `git clone`. If you deployed your app
-  in any other way (GitLab / GitHub integration, archive, ...) the changes made
-  to your code won't be available.
+  Only Git operations will be available via `git clone`.
+  If you deployed your app in any other way (GitLab / GitHub integration, archive, ...)
+  the changes made to your code won't be available.
 {% endnote %}
 
+#### Known Issue with Master Branch
+
+If you have the following message when doing a `git clone`:
+```
+warning: remote HEAD refers to nonexistent ref, unable to checkout.
+```
+
+It means that you're using the `master` branch. Since the default branch is now
+`main`, `git` doesn't know that you're using `master` branch instead of `main`.
+
+You need to manually checkout to the right branch inside the Git repository with:
+`git checkout master` to retrive the content of the repository.
+
+Or if you want to avoid this error when doing the `git clone` command you can
+add `--branch master` argument, like this:
+```bash
+git clone --branch master git@ssh.osc-fr1.scalingo.com:my-app.git
+```
+
```

Comment:
Should we add a clarification to indicate that `osc-fr1` should be replaced by `osc-secnum-fr1` if the client uses this region?

Reply:
I we do here, i could do that on all our documentation. Not sure if i want to change this here, because we have the same pattern everywhere

Reply:
Instead i will propose to add the list of SSH endpoints at the end of the page. We already have it on some other pages but this would be nice to have it in this page too